tools/kallisto_calc_expression.cwl

```diff
@@ -33,6 +39,8 @@ inputs:
   reads1: File
   reads2: File
```

Comment:
Shouldn't this be optional as well?

Reply:
Most likely, I was not too concerned with getting Kallisto to run properly since we are not using those results in the Splice project.